recipes/deblur/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set version = "1.0.4" %}
+
+package:
+  name: deblur
+  version: {{ version }}
+
+source:
+  url: https://github.com/biocore/deblur/archive/{{ version }}.tar.gz
+  sha256: bb43fefbf9f01db53a1f3bdff755c006a22717f976c8fa3fa242f74cb85380f4
+
+build:
+  number: 0
+  skip: True # [py27,py34]
+
+requirements:
+  build:
+    - python 3.5*
+    - biom-format
+    - setuptools
+    - scikit-bio >=0.5.0
+    - vsearch ==2.0.3
+    - mafft ==7.310
+    - sortmerna ==2.0
+    - biom-format
+    - h5py
+    - click
+  run:
+    - python 3.5*
```

Comment:
Please remove the python pinning here and only use the skipping in build section.